tests/test_request_stream.py

```diff
@@ -40,6 +42,53 @@ def get(self, request):
     assert response.text == data
 
 
+def test_request_stream_100_continue(app):
+
+    class SimpleView(HTTPMethodView):
+
+        @stream_decorator
+        async def post(self, request):
+            assert isinstance(request.stream, StreamBuffer)
+            result = ""
+            while True:
+                body = await request.stream.read()
+                if body is None:
+                    break
+                result += body.decode("utf-8")
+            return text(result)
+
+    app.add_route(SimpleView.as_view(), "/method_view")
+
+    assert app.is_request_stream is True
+
+    request, response = app.test_client.post("/method_view", data=data, headers={"EXPECT": "100-continue"})
```

Comment:
Maybe parameterize the test with different capitalization and variation of the header?

Reply:
Sounds good